molgenis-data-vcf/src/main/java/org/molgenis/data/vcf/VcfRepository.java

```diff
@@ -177,6 +186,15 @@ public String apply(Allele allele)
 			{
 				val = null;
 			}
+			// if a flag field exists in the line, than this field is true, although the value is null
```

Comment:
But more to the point: A little bit up I see that it's possible to check the VcfMetaInfo for type FLAG. Are there also non-flag booleans that would need to be written as NULL rather than TRUE when absent?